integration-cli/docker_cli_rename_test.go

```diff
@@ -9,26 +9,18 @@ import (
 )
 
 func (s *DockerSuite) TestRenameStoppedContainer(c *check.C) {
-	runCmd := exec.Command(dockerBinary, "run", "--name", "first_name", "-d", "busybox", "sh")
-	out, _, err := runCommandWithOutput(runCmd)
-	if err != nil {
+	if out, _, err := dockerCmdWithError("run", "--name", "first_name", "-d", "busybox", "sh"); err != nil {
```

Comment:
In cases where you don't care about err and you just need to check for err not nil you could just use `dockerCmd`